cogs/slowmode.py

```diff
@@ -8,9 +8,29 @@
 
 import util
 
+import time
+
 HELPER_ROLE = os.getenv('HELPER_ROLE')
 MOD_ROLE = os.getenv('MOD_ROLE')
 
+CHANNELS = [190327462087884811, 766335071590023247]
```

Comment:
`SLOWMODE_CHANNELS`

Reply:
changed variable name to `SLOWMODE_CHANNELS`